rcl/include/rcl/time.h

```diff
@@ -24,9 +24,9 @@ extern "C"
 #include "rcl/types.h"
 #include "rcl/visibility_control.h"
 
-#define RCL_S_TO_NS(seconds) (seconds * (1000 * 1000 * 1000))
-#define RCL_MS_TO_NS(milliseconds) (milliseconds * (1000 * 1000))
-#define RCL_US_TO_NS(microseconds) (microseconds * 1000)
+#define RCL_S_TO_NS(seconds) ((int64_t)seconds * (1000 * 1000 * 1000))
+#define RCL_MS_TO_NS(milliseconds) ((int64_t)milliseconds * (1000 * 1000))
+#define RCL_US_TO_NS(microseconds) ((int64_t)microseconds * 1000)
```

Comment:
good question, I didn't see any part of the codebase using it for anything else than for `rcl_wait` that uses `int64_t` that's why I went for it.
I would naively assume that an implicit cast from `int64_t` to `uint64_t` would just work without problem  but maybe I'm wrong